routes/manager.php

```diff
@@ -88,5 +88,6 @@
 Route::get('sites/stats/table/{date_start}/{date_end}', [StatsController::class, 'publisherStats']);
 Route::get('sites/stats/table2/{date_start}/{date_end}', [StatsController::class, 'publisherStatsWithTotal']);
 
+Route::patch('classifications/{site_id?}', [ClassifierController::class, 'add']);
```

Comment:
This endpoint is already defined two lines below. Please, delete this line.

Reply:
exactly :)